meeting-notes/2018/2018-11-28--gui-and-in-web-browsers-weekly.md

```diff
@@ -0,0 +1,150 @@
+        - virtuous gui - ddc loop. - https://github.com/ipfs/team-mgmt/issues/752#issuecomment-441117613
+
+    - Next:
+        - User tests for Web UI
+        - Upgrade peers page to do IP proximity instead of geolocation
+        - Upgrade files page to show the infinite plane.
+
+- @terichadbourne
+    - Done: 
+        - Feedback on copy for the share files app https://github.com/ipfs-shipyard/ipfs-share-files/issues/41
+        - WIP: saving code state and lesson completion state in ProtoSchool https://github.com/ipfs-shipyard/proto.school/pull/86
+    - Next: 
+        - Improve state saving in ProtoSchool 
+        - Move ProtoSchool to its own org so chapters can have their own subpages
+
+
+- @hacdias
+  - Done:
+    - Fix some random IPFS Desktop errors:
+      - https://github.com/ipfs-shipyard/ipfs-desktop/pull/709
+      - https://github.com/ipfs-shipyard/ipfs-desktop/pull/708
+      - https://github.com/ipfs-shipyard/ipfs-desktop/pull/711
+      - https://github.com/ipfs-shipyard/ipfs-desktop/pull/697
+    - Add i18n to Desktop: https://github.com/ipfs-shipyard/ipfs-desktop/pull/706
+  - Next:
+    - Merge this to fix some errors: https://github.com/ipfs-shipyard/ipfs-desktop/pull/722
+    - Fix some more issues with Desktop
+    - Add a storybook
+    - Auto-release binaries on tags (talk about Jenkins VS Travis)
+
+### Discussion Highlights ✨
+
+- [IPFS Companion shipped with Web UI v2.2](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.6.2)
+- Initial poc&success with [exposing API of js-ipfs running inside of Service Worker to a third-party website](https://github.com/ipfs-shipyard/service-worker-gateway/issues/30) (different Origin) over iframe proxy.
+    - This is part of "window.ipfs 2.0 / fallback-lib" OKR, the goal is to get a better understanding of existing constraints across vendors in available Web APIs for creating alternative "API providers". 
+    - Remaining work is to check if multiple websites embedding such iframe will share instance/repository/storage. 
+    - If time allows, split @lidel's sandbox into comprehensible examples.
+- Origin Trial of [Signed HTTP Exchanges](https://github.com/ipfs/in-web-browsers/issues/121) should land next week.
+    - We are just waiting for HTTP headers to land - [ipfs/infrastructure/issues/453](https://github.com/ipfs/infrastructure/issues/453)
+- [Share App](https://github.com/ipfs-shipyard/ipfs-share-files) updates
+    - New copy for upload and download screen
+    - We will add i18n soon
+    - Discovery speed without local daemon needs some extra work
+        - There should be some UI indication that something is happening behind the scenes
+    - When that is resolved, we will swap Share link in Web UI to point at the Share App 
+- [Migration to CIDv1](https://github.com/ipfs/ipfs/issues/337): Adding data as CIDv0 and getting it by CIDv1
+    - PR with interop tests: [ipfs/interop/pull/46](https://github.com/ipfs/interop/pull/46)
```

Comment:
@kyledrake FYI tests from linked PR give a good overview where we are with "Adding data as CIDv0 and getting it by CIDv1" and what remains to be done